<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusinessObjectList</class>
 <widget class="QWidget" name="BusinessObjectList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>706</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>971</width>
     <height>671</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>690</width>
     <height>0</height>
    </size>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_debug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>741</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDetails">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>145</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>145</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>125</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Details View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSummary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>160</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Summary View</string>
   </property>
  </widget>
   <widget class="QLabel" name="lblLeaderboard">
     <property name="geometry">
       <rect>
         <x>380</x>
         <y>10</y>
         <width>400</width>
         <height>16</height>
       </rect>
     </property>
     <property name="minimumSize">
       <size>
         <width>400</width>
         <height>0</height>
       </size>
     </property>
     <property name="font">
       <font>
         <weight>75</weight>
         <bold>true</bold>
       </font>
     </property>
     <property name="text">
       <string>Leaderboard 1.0</string>
     </property>
   </widget>
  <zorder>label_debug</zorder>
  <zorder>tableWidget</zorder>
  <zorder>btnDetails</zorder>
  <zorder>btnSummary</zorder>
  <zorder>lblLeaderboard</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
